tpl/tplimpl/embedded/templates/_default/rss.xml

```diff
@@ -21,7 +21,7 @@
     <managingEditor>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</managingEditor>{{end}}{{ with .Site.Author.email }}
     <webMaster>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</webMaster>{{end}}{{ with .Site.Copyright }}
     <copyright>{{.}}</copyright>{{end}}{{ if not .Date.IsZero }}
-    <lastBuildDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</lastBuildDate>{{ end }}
+    <lastBuildDate>{{ now.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</lastBuildDate>{{ end }}
```

Comment:
So, `lastBuildDate` should signal change, so we cannot use `now`. But I agree that the current value is not correct -- the best we can do is to use `.Lastmod`.

Reply:
Now is a bit aggressive, it should indicate the last change of any of the entries, might be possible, let me play with it.

Reply:
As I said, `.Lastmod` is the value to use.

Reply:
It looks like pages are sorted to `desc date` by default, updated to:

```
    {{- with (index $pages 0) -}}
    <lastBuildDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</lastBuildDate>{{ end }}
    {{- end -}}
```

If that is not guaranteed we could the following lines after the pages have been selected:
```
{{- $pages = sort $pages "Date" "desc" }}
```

Reply:
> As I said, `.Lastmod` is the value to use.

Let me check that as well.

Reply:
https://github.com/gohugoio/hugo/issues/7310#issuecomment-981137769

Reply:
> > As I said, `.Lastmod` is the value to use.
> 
> Let me check that as well.

The `Lastmod` value is from the main page, not relevant to the last new entry to the feed.

Sometimes it could make sense to use the most recent Lastmod value of the all pages (e.g., when a path changed) but for that I would need to create a separate sorting of the included pages and limit to the first entry.

Reply:
> [#7310 (comment)](https://github.com/gohugoio/hugo/issues/7310#issuecomment-981137769)

Thanks @jmooring 

`.Site.LastChange` works well and is definitely the winner in performance, maybe not 100% accurate (as the pages in this feed might not have not changed) but I don't think that matters much.